### ISSUE TITLE
Client library

### ISSUE THREAD

**Issue by jml** [haskell-graphql/graphql-api#155]
It'd be nice to be able to write GraphQL clients in Haskell. This could quite possibly be a separate project.

**Comment by michalrus** [comment]
Yes, please! :pray: Otherwise, how do we write integration tests?

**Comment by charlescrain** [comment]
Is anyone aware of a haskell GraphQL client library? 

https://haskell-graphql-api.readthedocs.io/en/latest/tutorial/Introduction.html#running-queries 
Above link has an example for running queries but it's necessary to write the type out manually.
